tools/pyboard.py

```diff
@@ -507,19 +516,33 @@ def fs_get(self, src, dest, chunk_size=256):
                 if not data:
                     break
                 f.write(data)
+                written += len(data)
+                if progress_callback:
+                    progress_callback(written, src_size)
+            if progress_callback:
+                progress_callback(src_size, src_size)
```

Comment:
I think these two lines are unnecessary, the callback would have already been called with maximum size in the last iteration of the while loop (?).

Reply:
Yep. They were these to ensure progress bar is cleared, but indeed this should happen anyway.